src/StaticWebAssetsSdk/Tasks/JSModules/GenerateJSModuleManifest.cs

```diff
@@ -24,7 +24,7 @@ public class GenerateJSModuleManifest : Task
 
         public override bool Execute()
         {
-            var modules = JsModules.Select(StaticWebAsset.FromTaskItem).Select(s => s.ComputeTargetPath("", '/')).ToArray();
+            var modules = JsModules.Select(StaticWebAsset.FromTaskItem).Select(s => s.ComputeTargetPath("", '/', StaticWebAssetTokenResolver.Instance)).ToArray();
```

Comment:
I've looked at all the other instances of `ComputeTargetPath` this was the only one missing.